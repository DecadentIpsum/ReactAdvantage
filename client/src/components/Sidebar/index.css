.sidebar-close-button {
  position: absolute;
  right: 0px;
  top: 20px;
  border: none;
  background: none;
  color: rgba(179, 179, 171, 0.8);
  cursor: pointer; }

.ui-sidebar-close {
  display: none !important; }
