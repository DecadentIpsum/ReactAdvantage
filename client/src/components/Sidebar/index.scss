.sidebar-close-button {
    position: absolute;
    right: 0px;
    top: 20px;
    border: none;
    background: none;
    color: rgba(#B3B3AB, 0.8);
    cursor: pointer;
}

.ui-sidebar-close {
    display: none !important;
}