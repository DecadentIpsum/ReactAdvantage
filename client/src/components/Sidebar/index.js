import React, { Component } from 'react';
import { Sidebar as PrimeSidebar } from 'primereact/components/sidebar/Sidebar';
import './index.css';

export default class Sidebar extends Component {
    render() {
        return (
            <PrimeSidebar className="right-sidebar-menu" {...this.props}>
                {this.props.children}
            </PrimeSidebar>
        );
    }
}
