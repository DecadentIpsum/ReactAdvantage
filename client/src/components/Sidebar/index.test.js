import React from 'react';
import Enzyme, { shallow } from 'enzyme';

import Sidebar from './index';

it('renders correctly', () => {
    const component = shallow(<Sidebar />);

    expect(component).toMatchSnapshot();
});