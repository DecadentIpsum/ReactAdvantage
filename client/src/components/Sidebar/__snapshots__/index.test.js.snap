// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Sidebar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "baseZIndex": 0,
      "blockScroll": false,
      "children": undefined,
      "className": "right-sidebar-menu",
      "fullScreen": false,
      "id": null,
      "onHide": null,
      "onShow": null,
      "position": "left",
      "style": null,
      "visible": false,
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "baseZIndex": 0,
        "blockScroll": false,
        "children": undefined,
        "className": "right-sidebar-menu",
        "fullScreen": false,
        "id": null,
        "onHide": null,
        "onShow": null,
        "position": "left",
        "style": null,
        "visible": false,
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
